contracts/modules/cover/Cover.sol

```diff
@@ -286,6 +279,40 @@ contract Cover is ICover, MasterAwareV2, IStakingPoolBeacon, ReentrancyGuard, Mu
     emit CoverEdited(coverId, params.productId, segmentId, msg.sender, params.ipfsData);
   }
 
+  function expireCover(uint[] calldata coverIds) external onlyMember {
+    for (uint i=0; i < coverIds.length; i++) {
```

Comment:
one potential contract size optimization here would be to take as input only 1 cover id, instead of looping through this array. it's not ideal, but if we can keep the Multicall in, it's not that bad